### ISSUE TITLE
Transcription guidelines field: have an "ontology" of values

### ISSUE THREAD

**Issue by PonteIneptique** [HTR-United/schema#5]
This one has been in my head for quite a long time.

Right now, we have free text, which means it is not machine actionable. I'd like to have the ability to populate a list of acceptable value, such as `Resolved Abbreviation`, `Unresolved Abbreviation`, `Corrected Spelling`, `Original Spelling`, `Special Character used` (eg. MUFI), things like that. It would go alongside the transcription guidelines but would make the whole thing a little more machine actionable.

**Comment by gabays** [comment]
True. This complicated though. What people understand as "original" varies a lot: some people consider the dissimilation of _i_/_j_ as faithful to the original, some don't… `corrected` is somewhat ambiguous though, because you can
- correct a mistake in the document while transcribing
- "correct" the spelling, which is more a "linguistic normalisation" to me

Abbréviations and special characters make more sense to me because it is more factual

**Comment by PonteIneptique** [comment]
I think we can have many values, with descriptions, and `Dissimilation of i/j` for example is for example one. We "just" need to make a list, this is a first step.

**Comment by PonteIneptique** [comment]
I'd actually cover those. u/v i/j go together generally, s/S less. So I'd have an entry for these specifically

**Comment by PonteIneptique** [comment]
The idea is to make the catalog as machine actionable as possible. If it means somewhat fine-grained values, then we go for it.

**Comment by gabays** [comment]
_Corrected spelling / Original spelling / Normalized spelling ?_ No: the original spelling or the normalised spelling can both be corrected. I would go for:
- original/modified
- corrected/not corrected

**Comment by PonteIneptique** [comment]
I asked for a long answer with a more descriptive thingie (like the parenthesis I wrote up there, you can give examples :D )

**Comment by gabays** [comment]
I don't get of it would look like at the end with the parenthsis

**Comment by PonteIneptique** [comment]
So, could you elaborate on this one, maybe with an example ? I know I am tiresome :) 

> original/modified (i.e. as diplomatic as possible or editorial interventions)

**Comment by gabays** [comment]
corrected/not corrected
1.  _il arriuoiz dans l'vniuers Parisien_ -> not corrected (and original)
2.  _il arrivoiz dans l'univers Parisien_ -> not corrected (and modified)
3. _il arrivait dans l'univers parisien_ -> corrected (and modified)

**Comment by gabays** [comment]
With original/modified it try to bypass the whole complicated terminology about transcription and simplify the problem to: "what did you try to do?" be more faithful or more interpretative? The answer would not have to be super script

**Comment by PonteIneptique** [comment]
That's clearer, now I need other people to weigh in :)

**Comment by PonteIneptique** [comment]
@alix-tz What do you think ?

**Comment by alix-tz** [comment]
Since we will probably not get it right from the attempt, I think it could be interesting to create an escapable option in order for people who did something different than plainly copying the text to be able to signal it, if the options we propose don't include what they did. I've aways been told that when you do an annotation guide, you should have a joker token in order not to lose traces of unclear situations. 

---

Otherwise, would you consider adding something about the ponctuation specifically? Or would you consider it goes in the normalized vs. original categories?

---

I'm also thinking of projects which include printed and manuscript texts: sometimes 1 character can be used to signal "printed" when the transcriber only wanted to focus on the handwritten sections. Should we have an entry to designate that?

---

What about rare glyphs, like glyphs which would be specific to a writer? Situations could be: 
- original glyph (do you have a better idea of how to call them?) were not transcribed
- a generic character was used to signal the presence of an original glyph
- each original glyph was matched with a unique character

**Comment by alix-tz** [comment]
For the distinction "original / modified" and "corrected / not corrected", indeed, it's probably best to keep it simple, so I agree with the last proposition. 

It might already be complicated in some cases to find where the frontier between not corrected and corrected lays.

**Comment by PonteIneptique** [comment]
HI @alix-tz ,

Just to be clear, regarding

> I think it could be interesting to create an escapable option in order for people who did something different than plainly copying the text to be able to signal it,

I think this list of value would be optional, non enforcable. The original transcription free text value would remain the main field. This is just to allow, down the road, a better machine actionable way to treat transcription guidelines.

**Comment by gabays** [comment]
I agree with Alix: punctuation and spelling are two different things -- especially for medievalists.

**Comment by dstoekl** [comment]
We use e.g. Leyden plus similar annotation for inline additions and deletions.
Aleph Lamed ligature instead of single aleph lamed if it is ligatured in the original. Now with annotations in eScr this might get extended to other letter combinations via tags.
I think it would be useful to give also the character set and its distribution.

**Comment by PonteIneptique** [comment]
We still need some values for this cases. The thing is, we can add as many
as we want, as long as it is helpful. Distinction of u/v & i/j for example
can be a value

Le mer. 23 mars 2022 à 8:07 AM, Simon Gabay ***@***.***> a
écrit :

**Comment by Boenig** [comment]
Hello,

Thibault pointed out your discussion.
I have not yet fully worked through the discussion.
Both I would like to contribute some thoughts and point out solutions. From my point of view, the problem presents itself on several levels.
Level 1.
The problem that a lot of GT datasets from the same source can differ is a concern for me as well as for our project. 
Level 2.
In what way can existing digital texts, editions.... can also be converted into GT. To what measure is digital text different from the original? 

As I have taken from the discussion.
1. the characters, glyphs....
2. I have not yet found structures:
e.g. now with this template
![image](https://github.com/HTR-United/schema/assets/25387135/86563af1-246b-454a-afc6-c59d4ac8218e)

DefaultLine, or DropCapitalLine, DefaultLine?

From OCR-D side we have defined a level system. 
Which is generally and in particular defined in the OCR-D-GT Guidelines: https://ocr-d.de/en/gt-guidelines/trans/trLevels.html.
 
Regarding the characters... corresponding level related rule sets are generated based on the guidelines.
see: https://github.com/tboenig/gt-guidelines/tree/gh-pages/rules
